hr_employee_anniversary/__manifest__.py

```diff
@@ -0,0 +1,29 @@
+{
+    'name': "Employee's anniversary",
+
+    'summary': """
+        Define the anniversary date for employees.
+        """,
+
+    'description': """
+        Adds a employment anniversary field next to birthday field.
+    """,
+
+    'author': "BerrySoft MX, Odoo Community Association (OCA)",
+    'website': "https://github.com/OCA/hr",
+
+    # Categories can be used to filter modules in modules listing
+    # for the full list
+    'category': 'Generic Modules/Human Resources',
+    'version': '0.1',
```

Comment:
@MarkingSC IMO `'version': '0.1'`, -->` 'version': '12.0.1.0.0',` or If new module then ` 'version': '12.0.0.0.0',`

Reply:
Thanks, I will solve that.